dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -566,6 +567,36 @@
 			label = "ADC_3";
 			#io-channel-cells = <1>;
 		};
+
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020000 0x400>;
+			interrupts = <11 0>, <12 0>, <13 0>, <14 0>, <15 0>, <16 0>,
+						 <17 0>, <47 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x00000020>;
+			status = "disabled";
+			label = "DMA_1";
+		};
+
+		dma2: dma@40020400 {
+			compatible = "st,stm32-dma";
+			#dma-cells = <4>;
+			reg = <0x40020400 0x400>;
+			interrupts = <11 0>, <12 0>, <13 0>, <14 0>, <15 0>, <16 0>,
+						 <17 0>, <47 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x00000020>;
+			status = "disabled";
+			label = "DMA_2";
+		};
+
+		rng: rng@48021800 {
```

Comment:
Support for RNG should go into a separate commit, same as for DMA.

Reply:
> Support for RNG should go into a separate commit, same as for DMA.

How the commit should be marked then? (e.g.: `boards: nucleo_h745zi_q: dma: `and `boards: nucleo_h745zi_q: rng:` )?

Reply:
The functionality is not tied to a specific SoC (you are changing that for all H7 series).

I think something like: `dts: arm: st: h7: add rng peripheral support` would be correct

Reply:
> The functionality is not tied to a specific SoC (you are changing that for all H7 series).
> 
> I think something like: `dts: arm: st: h7: add rng peripheral support` would be correct

Okay. I'll provide changes later.

Reply:
Splitted into two commits (one for dma's and one for rng), as suggested.

Reply:
done